tests/data_validation/great_expectations/checkpoints/comic_title_validations.yml

```diff
@@ -23,7 +23,7 @@ validations:
   - batch_request:
       datasource_name: newsletter_automation_datasource
       data_connector_name: default_runtime_data_connector_name
-      data_asset_name: newsletter_automation.articles
+      data_asset_name: Comics
```

Comment:
@ajitava-git Were you able to test this change? I don't think data_asset_name can be changed like that. It refers to database_name.table_name. It is referred similarly in the expectation files as well.

Reply:
I verified and we can change the data asset name. I ran workflow against this branch and was able to get results.